src/middleware/index.js

```diff
@@ -44,12 +44,13 @@ const ACTIONS_TO_TRACK = ['CREATE_NEW_ACCOUNT','ADD_ACCESS_KEY',
 'SETUP_ACCOUNT_RECOVERY', 'RECOVER_ACCOUNT','REMOVE_ACCESS_KEY']
 
 if (process.env.MIXPANEL_TOKEN) {
-    mixpanel.init(process.env.MIXPANEL_TOKEN)
+    mixpanel.init(process.env.MIXPANEL_TOKEN, {'property_blacklist': ['$current_url']})
 }
 
 const analyticsMiddleware = store => next => action => {
     let details = {
-        pathname: window.location.pathname,
+        pathname: !window.location.pathname.includes('recover-with-link') ? 
```

Comment:
We can store `recover-with-link` as const, we try to do that with all path related functionalities, like here:
https://github.com/near/near-wallet/blob/master/src/utils/wallet.js#L12

Not relevant to this PR, but I think generally if we want to relay on URL's we should store all routes in a separate file as consts, and additionally an array of objects with properties: `path`, `component`, `exact`, so we could map through it in `Routing.js`. What do you think @vgrichina @Patrick1904 ?

Reply:
I agree, it makes sense to organize this a bit more as number of routes/urls grows

Reply:
OK, I added an issue, to be handled in the near future.